README.md

```diff
@@ -1,7 +1,128 @@
+functions, losses, metrics and other necessary components to achieve state of the art accuracy.
+
+There will be common shareable components between those 2 tasks, like model building blocks, etc. We
+will have them in the repository as public API as well.
+
+### Common components
+- Model building block (served as backbone for the application)
+  - Resnet 
+  - Resnet RS
+  - ResNeXT
+  - Efficient
+- Preprocessing layers
+  - CutMix
+  - MixUp
+  - CutOut
+  - RandAugment
+  - RandomErasing
+
+### Image Classification
+- Dataset to evaluate
+  - [Imagenet](https://www.tensorflow.org/datasets/catalog/imagenet2012)
+  - [Imagenet v2](https://www.tensorflow.org/datasets/catalog/imagenet_v2) (for testing)
+- Target Metric 
+  - Top1 Accuracy > 80% (Note that some existing SotA models might not achieve this, like 
+    original ResNet, but this serve as a guideline for newly added models)
+
+### Object Detection
+- Dataset to evaluate
+  - [Coco 2017](https://www.tensorflow.org/datasets/catalog/coco#coco2017)
+  - [Open image v4](https://www.tensorflow.org/datasets/catalog/open_images_v4)
+  - [PASCAL voc](https://www.tensorflow.org/datasets/catalog/voc)
+- Target Metric
+  - COCO metric average precision (AP)
+- Detection model
+  - MaskRCNN
+  - YOLO (v3 or v4)
+  - RetinaNet
+- Other related utilities
+  - **ROI** (region of interest) layer (faster RCNN and Mask RCNN)
+  - Bounding box utils
+  - **RPN** (Region Proposal Network) used by faster RCNN and mask RCNN for bounding box 
+    generation.
+
+## Overlapping between keras.application and Keras-CV
+As you might notice, some area we would like to focus has overlapping with existing keras.application,
+and we would like to clarify the reason of the potential code duplication here.
+
+keras.application has a strong API contract that we can't easily make any backward incompatible
+change. On the other hand, there are a few of behavior we actually would like to change within the keras-cv
+  - Fetch imagenet weights by default, which might not want to do for keras-cv.
+  - Have the image preprocessing logic outside the model. We want to include all the preprocessing
+    within the model, so that user won't need to call the extra preprocessing methods.
+  - Only end-to-end model are exposed as public API, and Keras CV would like to include more building 
+    blocks as well.
+  - Some legacy model architectures are no longer commonly used and can't reach SotA performance, which
+    we might want to deprecate.
+
+Once the keras-cv applications are mature enough, we would add a deprecation warning in the 
+keras.applications code.
+
+
+## Community contribution guideline
+We would like to leverage/outsource the Keras community not only for bug reporting or fixes, 
+but also for active development for feature delivery. To achieve this, we will have the predefined
+process for how to contribute to this repository.
+
+The ideal workflow will be:
```

Comment:
It would be really nice if we could standardize this section (I suppose it will be something like CONTRIBUTING.md) with  a baseline template:
https://github.com/tensorflow/community/pull/384

I suppose that having a few common section it could help our contributors to navigate quickly our repositories.

Reply:
Ack, will put a contribution.md in a follow up PR.

Reply:
Don't worry. 
After some of the last comments I think that we are more oriented to build a "Keras ecosystem" here so we don't need to enforce things that are proposed under the Tensorflow/community github organization for the Tensorflow Ecoystem.

But if you like or you want to give some inline review it will be appreciated.

I have also a `readme.md` template under review in the same repo:
https://github.com/tensorflow/community/pull/376